### PR TITLE
Dread: Final Dairon revision

### PR DESCRIPTION
**Pull request by XenoWars**
This PR should finish off the Dairon logic revision. It covers the rest of the rooms, but not many changes here.

- EMMI Zone Exit Northwest
- EMMI Zone Exit West
- EMMI Zone Exit Southwest
- Burenia Upper Transport Access
- EMMI Zone Exit South
- Storm Missile Gate Room
- Lake Puzzle Room

### CODE REVIEW

File: randovania/games/dread/json_data/Dairon.txt [hunk 1]
```diff
@@ -2797,28 +2785,34 @@ Extra - asset_id: collision_camera_033
...
       Trivial
   > Pickup (Missile+ Tank)
       All of the following:
           Speed Booster
           Any of the following:
               Lay Bomb or Lay Power Bomb
-              After Dairon - Storm Gate Speed Blocks Destroyed and Walljump (Beginner)
-  > Tile Group (BOMB)
-      Morph Ball
+              After Dairon - Storm Gate Speed Blocks Destroyed and Movement (Intermediate)
```
Comment (kirbymastah) [inline randovania/games/dread/json_data/Dairon.txt:2797]: I'm guessing there's space to charge a speedbooster from the water room in the water? Either way, I would add a comment on where the speedboost is charged for clarification

File: randovania/games/dread/json_data/Dairon.txt [hunk 2]
```diff
@@ -2909,26 +2896,33 @@ Extra - asset_id: collision_camera_035
...
-          Lay Cross Comb or Lay Power Bomb
+          Any of the following:
+              # Power Bombs require desyncing the beam blocks
+              Lay Cross Comb or Lay Power Bomb
   > Above Grapple Block
       All of the following:
-          Morph Ball and After Dairon - grapplepulloff1x2_003
-          Cross Bomb or Gravity Suit or Space Jump
+          Morph Ball and After Dairon - Lake Puzzle Grapple Block
+          Cross Bomb or Gravity Suit or Use Spin Boost
```
Comment (kirbymastah) [inline randovania/games/dread/json_data/Dairon.txt:2925]: How does spinboost reach the water tunnel? It looks like it's too far to reach

File: randovania/games/dread/json_data/Dairon.txt [hunk 3]
```diff
@@ -3096,19 +3082,17 @@ Extra - asset_id: collision_camera_039
   * Extra - right_shield_entity: {EMPTY}
   * Extra - right_shield_def: None
   > Dock to EMMI Zone Exit Southwest
-      All of the following:
-          Morph Ball
-          Grapple Beam or Spider Magnet or Space Jump or Walljump (Beginner)
+      Morph Ball
```
Comment (kirbymastah) [inline randovania/games/dread/json_data/Dairon.txt:3085]: Looking at this again, is morph ball necessary to get to the top? We could just replace this with sliding. My only concern is if it's not possible to jump out of the bottom-right pond without gravity, and thus walljump out.

**Review by kirbymastah — CHANGES_REQUESTED** [review]
Few comments

File: randovania/games/dread/json_data/Dairon.txt (hunk 1 above)
Comment (XenoWars) [reply randovania/games/dread/json_data/Dairon.txt:2797]: This is for charging the speed where the Storm Gate was, but its very tight.

File: randovania/games/dread/json_data/Dairon.txt (hunk 2 above)
Comment (XenoWars) [reply randovania/games/dread/json_data/Dairon.txt:2925]: You can just fall through the crumble blocks then use spin boost to grab the ledge

File: randovania/games/dread/json_data/Dairon.txt (hunk 3 above)
Comment (XenoWars) [reply randovania/games/dread/json_data/Dairon.txt:3085]: While you can, in fact, just use slide, I guess I didn't change it cause you can't get anywhere further without morph anyway. All paths forward are blocked by it (the morph ball launchers, need morph to grab the item in the shortcut room, the tunnel to CU Access, and the tunnel to Exit Northwest). Should it still be changed anyway?

**Review by kirbymastah — APPROVED** [review]

**Pull request merged**